string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-unused-vars */
+/* convert title into array,
+loop through array,
+
+if word is api, then concatenate API
+
+if word is javascript, then concatenate JavaScript
+
+if not index of 0 or follows a ':':
+    then return if word is conjunction, article, preposition
+
+making each first letter(word at index of 0) capitalized,
+concatenate resulted capitalized word to stringToReturn,
+return stringToReturn
+*/
+
+function titleCase(title) {
+  var arrayOfWords = title.split(' ');
+  var stringToReturn = '';
+  var arrayOfLowercaseWords = ['and', 'or', 'nor', 'but', 'a', 'an', 'the', 'as', 'at', 'by', 'for', 'in', 'of', 'on', 'per', 'to'];
+
+  for (var i = 0; i < arrayOfWords.length; i++) {
+    var word = arrayOfWords[i];
+
+    if (i === 0) {
+      if (word.toLowerCase() === 'javascript:') {
+        stringToReturn += 'JavaScript:';
+        continue;
+      }
+      if (word.toLowerCase() === 'javascript') {
+        stringToReturn += 'JavaScript';
+        continue;
+      }
+      if (word.toLowerCase() === 'api') {
+        stringToReturn += 'API';
+        continue;
+      }
+    } else {
+      if (word.toLowerCase() === 'javascript:') {
+        stringToReturn += ' JavaScript:';
+        continue;
+      }
+      if (word.toLowerCase() === 'javascript') {
+        stringToReturn += ' JavaScript';
+        continue;
+      }
+      if (word.toLowerCase() === 'api') {
+        stringToReturn += ' API';
+        continue;
```

Comment:
You don't need all these continute statements, you can switch if and else if statements to get the same outcome.